astro/src/content/docs/apis/_application-response-body-base.mdx

```diff
@@ -96,26 +96,40 @@ import Xmlsignaturec14nmethodValues from 'src/content/docs/_shared/_xmlSignature
     * `SlidingWindow` - the expiration is calculated from the last time the token was used.
     * `SlidingWindowWithMaximumLifetime` - the expiration is calculated from the last time the token was used, or until the <InlineField>maximumTimeToLiveInMinutes</InlineField> is reached. <span class="text-green-600">Available since 1.46.0</span>
   </APIField>
+  
   <APIField name={ props.base_field_name + ".jwtConfiguration.refreshTokenSlidingWindowConfiguration.maximumTimeToLiveInMinutes" } type="Integer" since="1.46.0">
     The maximum lifetime of a refresh token when using a <InlineField>refreshTokenExpirationPolicy</InlineField> of `SlidingWindowWithMaximumLifetime`.
   </APIField>
+
   <APIField name={ props.base_field_name + ".jwtConfiguration.refreshTokenTimeToLiveInMinutes" } type="Integer" since="1.2.0">
     The length of time in minutes the JWT refresh token will live before it is expired and is not able to be exchanged for a JWT.
   </APIField>
+  
+  <APIField name={props.base_field_name + '.jwtConfiguration.refreshTokenUsagePolicy'} type="String" since="1.17.0">
```

Comment:
Verified that this was added to the 1.17. I checked the release notes.

https://fusionauth.io/docs/release-notes/archive#version-1-17-0